recipes/perl-yaml/1.15/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: perl-yaml
+  version: "1.15"
+
+source:
+  fn: YAML-1.15.tar.gz
+  url: https://cpan.metacpan.org/authors/id/I/IN/INGY/YAML-1.15.tar.gz
+  md5: 17bd31310367173b411b31c09a755e3e
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - perl x.x
```

Comment:
Fixed, sorry. I was somehow in python land and wanted to make it mulit-perl versionish....